components/builder-originsrv/src/data_store.rs

```diff
@@ -2035,43 +1982,6 @@ impl DataStore {
     }
 }
 
-fn sync_origins(pool: Pool, mut route_conn: RouteClient) -> DbResult<EventOutcome> {
-    let mut result = EventOutcome::Finished;
-    let conn = pool.get()?;
-    let rows = &conn
-        .query("SELECT * FROM sync_origins_v1()", &[])
-        .map_err(DbError::AsyncFunctionCheck)?;
-    if rows.len() > 0 {
-        let mut request = sessionsrv::AccountOriginCreate::new();
-        for row in rows.iter() {
-            let aid: i64 = row.get("account_id");
-            let oid: i64 = row.get("origin_id");
-            request.set_account_id(aid as u64);
-            request.set_account_name(row.get("account_name"));
-            request.set_origin_id(oid as u64);
-            request.set_origin_name(row.get("origin_name"));
-            match route_conn.route::<sessionsrv::AccountOriginCreate, NetOk>(&request) {
-                Ok(_) => {
-                    conn.query("SELECT * FROM set_session_sync_v1($1)", &[&oid])
-                        .map_err(DbError::AsyncFunctionUpdate)?;
-                    debug!(
-                        "Updated session service with origin creation, {:?}",
-                        request
-                    );
-                }
-                Err(e) => {
-                    warn!(
-                        "Failed to sync origin creation with the session service, {:?}, {}",
-                        request, e
-                    );
-                    result = EventOutcome::Retry;
-                }
-            }
-        }
-    }
-    Ok(result)
-}
-
 fn sync_packages(pool: Pool, mut route_conn: RouteClient) -> DbResult<EventOutcome> {
```

Comment:
When we merge jobsrv then you can remove sync_packages, AND all of the underlying sync code in builder-db as well!

Reply:
do you want me to do that as part of this PR? I'm neck deep in it anyway